proto/storage/job.proto

```diff
@@ -16,6 +16,8 @@ option go_package = "github.com/diagridio/go-etcd-cron/storage;storage";
 message JobRecord {
   string name = 1;
   string rhythm = 2;
-  map<string, string> metadata = 3;
-  google.protobuf.Any payload = 4;
+  int64 startTimestamp = 3;  // time that the schedule starts
+  int32 repeats = 4;  // time total time the job should trigger
```

Comment:
Any reason not to use `32`? it is odd to not be consistent.

```suggestion
  int64 repeats = 4;  // time total time the job should trigger
```

Reply:
int64 is a requirement for time. While the repeats is not expected to be that large, making int64 overkill. Even int32 is already "too much".